src/pocketmine/block/BlockIds.php

```diff
@@ -87,7 +87,7 @@
 	const FURNACE = 61;
 	const BURNING_FURNACE = 62, LIT_FURNACE = 62;
 	const SIGN_POST = 63;
-	const DOOR_BLOCK = 64, WOODEN_DOOR_BLOCK = 64, WOOD_DOOR_BLOCK = 64;
+	const DOOR_BLOCK = 64, WOODEN_DOOR_BLOCK = 64, WOOD_DOOR_BLOCK = 64, OAK_DOOR_BLOCK = 64;
```

Comment:
This implies that this ID only applies for oak doors, or that using this ID always results in an oak door.